style.css

```diff
@@ -1,296 +1,520 @@
+}
+
+#languages {
+  padding: 0 0 40px;
+}
+
+button {
+  width: 125px;
+  height: 50px;
+  background-color: #fff;
+  color: #396df2;
+  border-radius: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  border: 1px solid #396df2;
+  transition: all 0.5s linear;
+}
+
+#resume-button {
+  transition: all 0.5s linear;
+}
+
+#resume-button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+#resume-button:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+button:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.image-wrapper {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.snapshoot {
+  width: 100%;
+}
+
+ul.poppins-regular {
+  list-style-type: none;
+  text-decoration: none;
+  padding-left: 0;
+}
+
+li.language-end {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  background-color: #f7f7f7f9;
+  padding: 4px 50px;
+  margin-left: 0;
+  border-radius: 10px;
+  padding-left: 0;
+  margin-bottom: 10px;
+  font-size: medium;
+}
+
+img.language-icon {
+  margin-left: 10px;
+  margin-right: 20px;
+}
+
+button#resume-button {
+  width: 150px;
+  height: 50px;
+  background-color: #fff;
+  color: #396df2;
+  border-radius: 12px;
+  font-size: 17px;
+  font-weight: 500;
+  border: 1px solid #396df2;
+}
+
+ul.social-media-icons {
+  padding-bottom: 30px;
+}
+
+h4.poppins-regular {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.arrow {
+  margin-right: 40px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 7px;
+}
+
+footer {
+  margin: 0;
+  background-image: url("./assets/bg\ image.png.png");
+  border-radius: 0 50px 0 0;
+  background-color: #6070ff;
+  padding: 20px;
+}
+
+.footer-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+}
+
+.form-container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 0 auto;
+}
+
+form {
+  width: 100%;
+}
+
+li.form-field {
+  width: 100%;
+}
+
+.form-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.text-field {
+  font-family: Poppins-regular, sans-serif;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  margin: 10px auto;
+  height: 40px;
+  border-radius: 10px;
+  width: 100%;
+  border: none;
+  padding-left: 10px;
+  padding-right: 0;
+}
+
+textarea {
+  font-family: Poppins-regular, sans-serif;
+  box-sizing: border-box;
+  padding-top: 10px;
+  height: 120px;
+  width: 100%;
+  border-radius: 10px;
+  padding-left: 10px;
+  margin: 10px 0;
+}
+
+.footer-container .poppins {
+  color: #fff;
+  text-align: center;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.footer-container .poppins-regular {
+  color: #fff;
+  text-align: center;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
```

Comment:
[OPTIONAL] Your project is looking good. :+1:  The only issue I have is that the design does not conform to the desktop figma design between screen width of 768px and 860px. Check the images below :point_down:  for illustration. Kindly ensure that it conforms to the dimensions of the desktop figma design for all screen size greater that 768px as stated in the project requirements. For your project to be responsive it has to conform to the figma designs in all screen sizes

**Project Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 17-28-30](https://user-images.githubusercontent.com/65233860/193324793-03f38b9b-d838-45ae-ae55-e3e14d4d9438.png)  |  ![Screenshot from 2022-09-30 17-29-40](https://user-images.githubusercontent.com/65233860/193324833-35a8bc62-66d4-4ca4-84db-055958890323.png)


**About Section**

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-30 17-28-14](https://user-images.githubusercontent.com/65233860/193324932-7661a99b-da21-4f2e-9a0d-4b4c7e45367e.png)  |  ![Screenshot from 2022-09-30 17-29-50](https://user-images.githubusercontent.com/65233860/193324980-4b7ea30a-a58d-4cd5-854a-dc7e8f3e69e6.png)

Reply:
Hi @jerryowusu ,
Thank you so much for approving my pull request, and for the suggestions given!